### ISSUE TITLE
ERROR    invalid co header size

### ISSUE THREAD

**Issue by PROGRAMADORRPL** [dashingsoft/pyarmor#1647]
When I try to obfuscate my script, I run into the following error. Initially, I considered that it could be attributable to the restrictions inherent to the trial version; However, when purchasing the fully licensed version, the problem persists.

```
(base) roger@DESKTOP-OGF0FUV Modificaciones % pyarmor gen ventana_app.py
INFO     Python 3.11.5
INFO     Pyarmor 8.4.6 (basic), 00000, xxxxx
INFO     Platform darwin.x86_64

```

**Comment by jondy** [comment]
Please run it with debug option and check the log
```
pyarmor -d gen xxxx.py
```

It will print the problem function name and line no, if possible, patest this function source here, or ignore it by `pyarmor cfg exclude_co_names`

**Comment by PROGRAMADORRPL** [comment]
Let me provide you with a more detailed explanation. The application consists of two windows: a registration window and an application window. The problem lies in the function of the script that spans the application window. This function encompasses the entire application and extends to approximately 7,000 lines of code. The specific anomaly has been identified in the function that handles the application window.

**Comment by jondy** [comment]
I don't care about what the function does, it could not help to fix this issue. I only need one script could reproduce this issue in my side.

**Comment by jondy** [comment]
If one function which uses too many cell variables will result in this issue. If it is in this case, please waiting for next release.

**Comment by jondy** [comment]
Please verify it with pre-release
https://pyarmor.dashingsoft.com/downloads/temp/pyarmor-8.4.7.zip

**Comment by PROGRAMADORRPL** [comment]
The problem is in version 8.4.6. With the new version it works perfectly. Thanks for everything.

**Comment by jondy** [comment]
Fixed in v8.4.7
